nRF Toolbox/MainScreen/MainRouter.swift

```diff
@@ -77,6 +77,12 @@ extension DefaultMainRouter: ServiceRouter {
         }
         self.splitViewController.showDetailViewController(viewController, sender: self)
     }
+    
+    func showLinkController(_ link: LinkService) {
+        let webViewController = WebViewController(link: link)
+        let nc = UINavigationController.nordicBranded(rootViewController: webViewController)
+        self.splitViewController.showDetailViewController(nc, sender: self)
```

Comment:
Minor: I think the 'self' prefix is not necessary here.

Reply:
(Same comment applies throughout the rest of the code)

Reply:
It's a habit. 
Removed.